endpoints/bahmni/input-validation.json

```diff
@@ -1 +1,549 @@
-{}
+{
```

Comment:
@Zooloo2014 When we don't have a `required` rule, do we default to false?

Reply:
That's for the validator logic to decide. 
But I think using the principle that "whatever is not defined, is defaulted" is a good principle to use , because it will require less specification on the schema files.